src/pages/orgs/[organizationId]/forks/[forkId].tsx

```diff
@@ -54,7 +57,7 @@ const SingleFork = (
   } = trpc.repos.listMirrors.useQuery(
     {
       orgId: organizationId as string,
-      forkName: fork?.name as string,
+      forkName: fork?.name ? fork?.name : '',
```

Comment:
```suggestion
      forkName: fork?.name ?? '',
```

Prefer nullish coalesce since it's the same prop we're checking and using